styles/style.css

```diff
@@ -0,0 +1,52 @@
+* {
+    margin: 0;
+    font-family: "Roboto", sans-serif;
+}
+.text {
+    padding-top: 120px;
+}
+.container-one {
+    height: 183px;
```

Comment:
It's a bad practice to use height without real reasons, please try to avoid this, or if you can't, I recommend to use min-height